include/psen_scan_v2/scanner_controller.h

```diff
@@ -216,7 +216,7 @@ void ScannerControllerT<TCSM, TUCI>::sendStopRequest()
 {
   // Before we send the stop request, we need to ensure that an potentially running start()-loop is stopped.
   stopStartReplyWatchdog();
-
+  data_udp_client_.close();
```

Comment:
Now close() is called twice on the Client? Is this safe?

Reply:
Should be no problem